### PR TITLE
small recommendations

### PR DESCRIPTION
**Pull request by clearsab**
### Description of Change
Adding few more beaches
- An enhancement to our core product :smiley:
4. Morjim
5.  Arambol

To Eat
4. Pani puri and others

## To Explore
1. Night life at Calangute beach
2. Shopping at flea Market

This links #27 

/cc: @githubteacher

### CODE REVIEW

File: goa.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Things to do
+## Beaches
+1. Candolim
+2. vagator
+3. Calangute
```
Comment (clearsab) [inline goa.md]: What about other places?

File: goa.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Things to do
+## Beaches
+1. Candolim
+2. vagator
```
Comment (clearsab) [inline goa.md]: V not v

Comment (clearsab) [reply goa.md]: ok

File: goa.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Things to do
+## Beaches
```
Comment (katjalinke) [inline goa.md:2]: which one is best?

Comment (clearsab) [reply goa.md:2]: All are good, but the ones in North Goa are really the best,
I found Arambol beach not mentioned in the list to be the best

**Review by githubteacher — APPROVED** [review]
A beach sounds nice right now! 👍

File: goa.md (hunk 3 above)
Comment (clearsab) [reply goa.md:2]: @katjalinke does it look good :+1:

**Pull request merged**